app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2 class="container_form">Sign up</h2>
+
+<div class="container_form">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field form_input ">
+    <%= f.text_field :name, placeholder: "Name", autocomplete: "name", autofocus: true %>
+  </div>
+
+  <div class="field form_input ">
+    <%= f.email_field :email, placeholder: "Email", autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field form_input ">
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, placeholder: "Password", autocomplete: "new-password" %>
+  </div>
+
+  <div class="field form_input ">
+    <%= f.password_field :password_confirmation, placeholder: "Confirm password", autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions form_input">
+    <%= f.submit "Sign up" , class: "button"%>
+  </div>
```

Comment:
- [ ] Great effort in implementing the `confirmation` feature! 👏 However, there's a minor flow issue that needs addressing. 

Currently, after a user has successfully created an account, they are redirected to the home page of the app. This isn't aligned with the intended flow. 

### Here's the recommended flow:

1. On a successful `sign_up`, users should be directed to a dedicated `UI` that asks them to `confirm` their email. This step is crucial for email verification.

2. After users have successfully confirmed their email, that's when you can redirect them to the home page or the login page for them to sign into their account.

This adjustment ensures a smoother and more secure onboarding experience for users.
Happy coding! 😊🚀🍀

Reply:
@Strangeal Hi Justice, please review this video where I show that this redirection is already in place https://www.loom.com/share/713ef1b2e1b7409eb3b1a9a77c188d97?sid=8582e224-d740-4ea8-a3f5-515653d91a2c